R/mash_wrapper.R

```diff
@@ -470,16 +470,59 @@ load_multitrait_R_sumstat <- function(
   }
 
   merge_matrices <- function(matrix_list, value_column, id_column = "variants",
-                             remove_any_missing = FALSE) {
-    # Convert matrices to data frames
-    df_list <- lapply(seq_along(matrix_list), function(i) {
+                             remove_any_missing = FALSE, ld_ref = ld_ref, allele_flip = TRUE) {
```

Comment:
`ld_ref` cannot default to a variable -- this can be considered a programming error. If this is required input, you can move it up to before `id_column` and make it required without default value.